### ISSUE TITLE
Unable to add Tinker's books to the Akashic Tome

### ISSUE THREAD

**Issue by MuteTiefling** [Vazkii/AkashicTome#68]
Simply doesn't go: 

![image](https://user-images.githubusercontent.com/9543430/120876522-252a9c00-c57f-11eb-8134-7243a6d877ad.png)

AkashicTome-1.4-16
TConstruct-1.16.5-3.0.4.171

Forge: 36.1.16

https://github.com/SlimeKnights/TinkersConstruct/issues/4395
https://github.com/NillerMedDild/Enigmatica6/issues/2351

**Comment by NielsPilgaard** [comment]
Possible fix going forward, from the Tinkers dev:

> 
> 
> I have tagged all my books under the item tag `forge:books/guide` and `forge:books`. Suggest to the other mods they should be using a tag check instead of a registry ID string match, way too many false positives in experience with registry ID check

https://github.com/SlimeKnights/TinkersConstruct/issues/4395#issuecomment-826214333

**Comment by MuteTiefling** [comment]
Tiny update. I hadn't realized that items needed to be whitelisted when I made this report. However, I've gone back and whitelisted each book explicitly and they still don't go in.

**Comment by steelfoot** [comment]
I'm having the same problem, with a slight twist:  I can add exactly _one_ of the TiCo manuals to the Akashic Tome at once.  The others will not be allowed to craft into the Akashic Tome until the added manual is removed.  I would have added this as a separate issue, but I believe the two are related or possibly even the same issue.

**Comment by SenojLuap** [comment]
I'm also seeing this while playing the ATM 6 modpack

**Comment by LemonShaped** [comment]
Pretty sure this is a duplicate of #53.

**Comment by aaronhowser1** [comment]
![image](https://user-images.githubusercontent.com/15698722/127217897-c4f1fa03-89db-4f5b-9f89-aa51e6270875.png)
```
"Whitelisted Items" = ["roots:runedtablet", "opencomputers:tool:4", "immersiveengineering:tool:3", "integrateddynamics:on_the_dynamics_of_integration", "theoneprobe:probenote", "evilcraft:origins_of_darkness", "draconicevolution:info_tablet", "charset:tablet", "antiqueatlas:antique_atlas", "occultism:dictionary_of_spirits", "tconstruct:materials_and_you", "tconstruct:puny_smelting", "tconstruct:mighty_smelting", "tconstruct:tinkers_gadgetry", "tconstruct:fantastic_foundry"]
```

It also does have that tag:
![image](https://user-images.githubusercontent.com/15698722/127217952-8c681d7b-b5e4-4124-b72d-ef3411d3a907.png)

**Comment by vizthex123** [comment]
hope this gets fixed in the 1.18 version of the mod

edit; I found that there's a whitelist in the mod config, you can add the tconstruct books to that for now (and hopefully Vazkii fixes this soon)

**Comment by eyeonus** [comment]
> Possible fix going forward, from the Tinkers dev:
> 
> > I have tagged all my books under the item tag `forge:books/guide` and `forge:books`. Suggest to the other mods they should be using a tag check instead of a registry ID string match, way too many false positives in experience with registry ID check
> 
> [SlimeKnights/TinkersConstruct#4395 (comment)](https://github.com/SlimeKnights/TinkersConstruct/issues/4395#issuecomment-826214333)

That does seem like a good idea.

Regarding this issue, in All the Mods 6 in MC1.16.5 with Akashic Tome v1.4-16, I've added the tinker's construct books to the "Whitelisted Items" list, and the Tome still will not allow any of them to be added.

To be clear:
```
"Whitelisted Items" = ["roots:runedtablet", "opencomputers:tool:4", "immersiveengineering:tool:3", "integrateddynamics:on_the_dynamics_of_integration", "theoneprobe:probenote", "evilcraft:origins_of_darkness", "draconicevolution:info_tablet", "charset:tablet", "antiqueatlas:antique_atlas", "occultism:dictionary_of_spirits", "tconstruct:materials_and_you", "tconstruct:puny_smelting", "tconstruct:mighty_smelting", "tconstruct:tinkers_gadgetry", "tconstruct:fantastic_foundry"]
```
But:
![image](https://user-images.githubusercontent.com/1733600/209633899-53e26680-45da-418c-b69e-8863c1a0ca44.png)
